### PR TITLE
Adding a web development project: Cryptopia [gssoc23] 

### LINKED ISSUE #2999: Adding a web development project: Cryptopia [gssoc23]
**Describe the project you want to add with tech stack**
This is a website named Cryptopia, this displays real-time trading value of the crypto. Various APIs are used in this project 
TECH STACK- HTML, JAVASCRIPT, CSS.

**Screenshots (optional)**

Screenshot (252)

Screenshot (253)

Screenshot (254)

Screenshot (255)

**Additional context (optional )**
Please add the tag of gssoc23

### PR DESCRIPTION
**Pull request by thevirengarg**
# Title and Issue number 

Web Development project: Cryptopia [gssoc23]
Previously, I opened a PR for the same issue, but i was asked to do some improvements so this is an updated PR. Previous PR #3013 

Close #2999 

# Checklist:

- [x] I have mentioned the issue number in my Pull Request.
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`

https://github.com/pranjay-poddar/Dev-Geeks/assets/111726483/26172041-086e-4b42-b21a-1b9afd9b0d49

Please add the tag of gssoc23.

### CODE REVIEW

**Comment by thevirengarg** [comment]
@MohitGupta121 @pranjay-poddar Please review this PR.

**Comment by thevirengarg** [comment]
@MohitGupta121 I have removed all images from this PR

File: Frontend-Projects/Cryptopia/assets/images/chart-1.svg
Comment (MohitGupta121) [inline Frontend-Projects/Cryptopia/assets/images/chart-1.svg:1]: This is still there

File: Frontend-Projects/Cryptopia/assets/images/coin-4.svg
Comment (MohitGupta121) [inline Frontend-Projects/Cryptopia/assets/images/coin-4.svg:1]: this one

File: Frontend-Projects/Cryptopia/fevicon.jpeg
Comment (MohitGupta121) [inline Frontend-Projects/Cryptopia/fevicon.jpeg:1]: this one

**Review by MohitGupta121 — CHANGES_REQUESTED** [review]
@thevirengarg you have to remove the image from PR, make sure you replace the src path of the image with that link of the image. So that they get load not just remove.

**Pull request closed without merging**